File: apps/s-2-admin/src/api/eventTypeCustomInput/EventTypeCustomInputUpdateInput.ts
import { EventTypeWhereUniqueInput } from "../eventType/EventTypeWhereUniqueInput";

export type EventTypeCustomInputUpdateInput = {
  eventType?: EventTypeWhereUniqueInput;
  label?: string;
  type?: "TEXT" | "TEXTLONG" | "NUMBER" | "BOOL";
  required?: boolean;
  placeholder?: string;
};

File: apps/cactus-demo-dotnet/src/APIs/EventTypeCustomInput/Dtos/EventTypeCustomInputUpdateInput.cs
using CactusDemoDotnet.Core.Enums;

namespace CactusDemoDotnet.APIs.Dtos;

public class EventTypeCustomInputUpdateInput
{
    public int? Id { get; set; }

    public int? EventType { get; set; }

    public string? Label { get; set; }

    public TypeEnum? Type { get; set; }

    public bool? Required { get; set; }

    public string? Placeholder { get; set; }
}
